.mm/pyre.lib.tests

```diff
@@ -42,8 +42,12 @@ tests.pyre.lib.memory.constmap_oob.pre := tests.pyre.lib.memory.map_write
 tests.pyre.lib.grid.grid_mmap_set.pre := tests.pyre.lib.grid.grid_mmap_sanity
 tests.pyre.lib.grid.grid_mmap_get.pre := tests.pyre.lib.grid.grid_mmap_set
 
-# exclude the tensor tests (require c++20)
+# conditionally exclude tensor tests that require c++20 or greater
+# NOTE:
+#  - remove the check when c++20 becomes the minimum required
+#  - in the meantime, add more versions as they are supported
 pyre.lib.tests.drivers.exclude += \
+${if ${findstring $($(compiler.c++).std.c++20),$(pyre.lib.tests.c++.flags)},,\
```

Comment:
Does this mean that these tests are active if the compiler supports C++20? If yes, then I will implement the same behavior in CMake. Right now, CMake adds these tests to the pile only if explicitly requested by the user, regardless of the compiler support.

Reply:
`mm` has a (slightly awkward but functional} way of checking the c++ standard requested by the user. is there similar functionality in `cmake`?